airgun/entities/contenthost.py

```diff
@@ -55,12 +56,39 @@ def execute_package_action(self, entity_name, action_type, value):
         view.progressbar.wait_for_result()
         return view.read()
 
+    def execute_module_stream_action(self, entity_name, action_type, module_name, stream_version):
+        """Execute remote module_stream action on a content
+
+        :param entity_name: content host name to remotely execute package
+            action on
+        :param action_type: remote action to execute. Can be one of 5: 'Enable', 'Disable'
+            'Install', 'Update', 'Remove', 'Reset' and 'Customize'
+        :param module_name: Module Stream name to remotely
+            install/upgrade/remove (depending on `action_type`)
+        :param stream_version: this uniduely identifies the module with Stream Version
+
+        :return: Returns a dict containing job status details
+        """
+        view = self.navigate_to(self, 'Edit', entity_name=entity_name)
+        view.module_streams.search('name = {} and stream = {}'.format(module_name, stream_version))
+        view.module_streams.table.row(
+            name=module_name, stream=stream_version)['Actions'].fill(action_type)
+        view = JobInvocationStatusView(view.browser)
+        view.wait_for_result()
+        return view.read()
+
     def search_package(self, entity_name, package_name):
         """Search for specific package installed in content host"""
         view = self.navigate_to(self, 'Edit', entity_name=entity_name)
         view.packages_installed.search(package_name)
         return view.packages_installed.table.read()
 
+    def search_module_stream(self, entity_name, module_name):
```

Comment:
why you omitted to add that status='All'?
perhaps it should be
```python
search_module_stream(self, entity_name, module_name, status='All'):
   ...
   view.module_streams.search(module_name, status=status)


```

Reply:
Thanks for remembering me! will add that